src/ApiIncrementalChecker.ts

```diff
@@ -115,10 +145,6 @@ export class ApiIncrementalChecker implements IncrementalCheckerInterface {
   }
 
   public getLints(_cancellationToken: CancellationToken) {
-    if (!this.linterConfig) {
```

Comment:
Does the removal of this speedy return mean that users who are not using tslint pay a performance tax as we still check for `tslint.json`s?

If so I'd like to try and avoid this

Reply:
as far as I can see, the answer is no, because `getLints` method get called only if `hasLinter` returns `true`
<img src="https://yadi.sk/i/5qqeQIw1z3s-IQ" width="600" />

Reply:
but still, I guess it never hurts to be more explicit. Fixed it

Reply:
So it looks like you're right - it does appear to be unnecessary. You had it right first time. 😁

Could you undo that - there's no point checking something a second time when it's already been checked.

I think then we'll be good to merge. Thanks for all your work!

Reply:
you got it!